### PR TITLE
How to Contribute to Ember

### PR DESCRIPTION
**Pull request by duggiefresh**
Some wordz and thingz and photoz

### CODE REVIEW

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+ever improving wave has been a wild, yet relatively safe ride. I've been working
+with Ember before it was cool, and can share my personal experience
+with helping out the community.
+
+![](https://i.imgur.com/Y2Zr7NA.jpg)
+
+### Help Answer Questions
+
+The [Ember community page][ember-community-page] lists various places where discussions,
+questions, and users can chat. However, it should be noted that from my experience,
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: This probably needs some rewording: "where discussions, questions, and users can chat" -> "where users can chat, discuss, or ask questions" perhaps.

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+IRC and discussion forum do not receive as much traffic as the [Slack community][ember-slack].
+
+I recommend using the Slack community as the number one resource for asking and
+answering questions. There are numerous channels that range from local user groups to
+testing. I suggest examining the channels you'd like to belong to, and engage with
+fellow users.
+
+The `#-help` channel is a great place to Ember-related ask questions.
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: "Ember-related ask questions" -> "ask Ember-related questions"

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+Furthermore, there are plently of opportunities to help your fellow developers
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: It sounds like you're repeating yourself.

> There are numerous ways to contribute to open source software. You can do one of the following:
1. **Help answer questions**

-

> The Ember community page lists various places where discussions, **questions**, and users can chat.

-

> I recommend using the Slack community as the number one resource for asking and **answering questions**.

-

> there are plently of opportunities to **help your fellow developers with their questions**.

Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: typo: `s/plently/plenty/`

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+with their questions.
+
+![](https://i.imgur.com/Hxw1qtd.png)
+_A real life enactment of someone helping another person in the Ember Slack community_
+
+### Write articles and improve learning resources
+
+I have read my fair share of articles based upon web development (I have stopped
+reading any articles about JavaScript fatigue though). I imagine many us have
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: `s/many us/many of us/`

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+learned a great deal from articles detailing one cool trick by a seasoned developer,
+or an article describing the pros and cons of given programming language.
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: "of **a** given programming language"

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+Just like any other open source project, Ember and its associated libraries, receive
+a [high number issues][ember-issues]. Luckily, we can help out!
+
+Let's say you happen to run into a "bug", what should you do next? First, search through
+the existing issue tracker for the given project, and [attempt to find your issue][obligatory-xkcd].
+If you find your issue, add a comment!
+
+Thorough comments on a particular issue can help tremendously. If you see a comment, and
+are experiencing the same issue, please don't bombard the maintainers with a "+1" comment.
+Instead, offer a something more insightful. You can attach an [Ember Twiddle][ember-twiddle]
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: "offer **~~a~~** something"

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+in an effort to recreate the issue, and state your expectation vs. reality.
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: Maybe replace "reality" with "actual behavior".

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+
+If you can't find an existing issue that replicates your issue, create a new one.
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: Repetition of "issue": "that replicates **yours**" would be better.

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+Ember actually has a [good recommendation on issue reporting][ember-issue-reporting].
+Following these steps helps to ensure that your issue will can be recreated and corrected
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: "**~~will~~** can be"

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+### Contribute Code
+
+Submitting [pull requests to Ember libraries is straightforward][ember-pr-guide].
+There is a channel within the Slack community called `dev-ember` that is reserved
+for the discussion of "development on Emberjs itself."
+
+So long as you follow the aforementioned guidelines, submitting a PR is an easy
+process.
+
+Just like any other codebase, Ember libraries can be imtimidating at first, however
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: typo: `s/imtimidating/intimidating/`

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+with enough persistence, you'll be cooking in no time! In addition, "contributing
+code" can take the form of [contributing documenation][ember-issues-docs].
+
+Lastly, if you stick around, the maintainers occassionally ask for community help,
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: typo: `s/occassionally/occasionally/`

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 12]
```diff
@@ -0,0 +1,177 @@
...
+There are plenty of meetups to join, and plenty of ways to contribute.
+Submitting a talk - whether it is a conference proposal or a lightning talk -
+will allow you to broadcast your experiences with Ember, and help generate new discussions.
+Secondly, I enjoy attending local meetups and conferences as it gives me the chance to
+place faces with GitHub, Twitter, and Slack handles.
+
+## Fin
+
+Hope you found this article helpful! And before you go, our entire blog is open sourced,
+so if you find a typo or want to make a suggestion, [please feel free][reefpoints]!
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: "if you find a typo" -- I think you need to enable spell checking :trollface: .

Comment (duggiefresh) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: 🚎 :trollface:

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+
+![](https://i.imgur.com/Hxw1qtd.png)
+_A real life enactment of someone helping another person in the Ember Slack community_
+
+### Write articles and improve learning resources
+
+I have read my fair share of articles based upon web development (I have stopped
+reading any articles about JavaScript fatigue though). I imagine many of us have
+learned a great deal from articles detailing one cool trick by a seasoned developer,
+or an article describing the pros and cons of given a programming language.
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: "of given a programming language" -> "of a given programming language"

File: source/posts/2016-09-30-how-to-contribute-to-ember.md [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+Submitting [pull requests to Ember libraries is straightforward][ember-pr-guide].
+There is a channel within the Slack community called `dev-ember` that is reserved
+for the discussion of "development on Emberjs itself."
+
+So long as you follow the aforementioned guidelines, submitting a PR is an easy
+process.
+
+Just like any other codebase, Ember libraries can be intimidating at first, however
+with enough persistence, you'll be cooking in no time! In addition, "contributing
+code" can take the form of [contributing documenation][ember-issues-docs].
```
Comment (brzpegasus) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: `s/documenation/documentation/`

Comment (duggiefresh) [inline source/posts/2016-09-30-how-to-contribute-to-ember.md]: I heard Emacs has spell checking

**Comment by duggiefresh** [comment]
Updated @brzpegasus! Thanks for the eagle 👀

**Comment by rsocci** [comment]
LGTM @duggiefresh!

**Pull request merged**